src/chainparams.cpp

```diff
@@ -200,7 +208,7 @@ class CRegTestParams : public CTestNetParams {
         consensus.nMajorityEnforceBlockUpgrade = 750;
         consensus.nMajorityRejectBlockOutdated = 950;
         consensus.nMajorityWindow = 1000;
-        consensus.powLimit = uint256S("7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff");
+        consensus.powLimit = uint256S("0x7fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"); // ~uint256(0) >> 1;
```

Comment:
Chose to make it uniform. If we do uint256S hex notations prefixed with 0x everywhere, then also here?